### ISSUE TITLE
running tests in parallel -what options to specify to view the responses

### ISSUE THREAD

**Issue by kthulsidoss** [postmanlabs/newman#1712]
Dear PostMan Support , greetings. I have been trying to use the parallel runner to run test parallely .I used the support mentioned here https://github.com/postmanlabs/newman/blob/develop/examples/parallel-collection-runs.js . There is a parallel issue albeit not a similar one here Feature request: Run test in parallel #744.  
I need to check the responses that are sent out , what do i specify so that i can check the responses 
(BTW i store the response in the environment body) ,here is what i see in the command prompt 
C:\npm\node_modules>node parallel-collection-runs.js
Defect74971GetApiDataCombinedUsedByISIN ran successfully.
Defect74971GetApiDataCombinedUsedByISIN ran successfully.
Defect74971GetApiDataCombinedUsedByISIN ran successfully.

Appreciate your quick response as this is needed to verify a critical defect in our product.

**Comment by numaanashraf** [comment]
@kthulsidoss Could you try adding `console.log(pm.response.text())` in the test script? That should log to the stdout.

**Comment by kthulsidoss** [comment]
@numaanashraf  sorry no output received
C:\npm\node_modules>node parallel-collection-runs.js
Defect74971GetApiDataCombinedUsedByISIN ran successfully.
Defect74971GetApiDataCombinedUsedByISIN ran successfully.
Defect74971GetApiDataCombinedUsedByISIN ran successfully.

C:\npm\node_modules>

i added the code to the script postman.setEnvironmentVariable("JSONResponse", pm.response.text())

console.log(pm.response.text())

**Comment by codenirvana** [comment]
@kthulsidoss You are using Newman as a library, to get `console.log` in stdout you need to use CLI reporter.

In your `newman.run` [options](https://github.com/postmanlabs/newman/blob/develop/examples/parallel-collection-runs.js#L16) add `reporters: 'cli'` to enable CLI reporter.

**Comment by kthulsidoss** [comment]
it does work ,thanks all, how ever there is a new issue which i'm reporting
